### PR TITLE
Book calendar

### PR DESCRIPTION
**Pull request by ghost**
We can see the date book in the calendar

### CODE REVIEW

**Review by LauraCourquin — APPROVED** [review]
It's good for me !

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: backend/src/controllers/CalendarController.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      });
+  };
+
+  static add = (req, res) => {
+    const calendar = req.body;
+
+    // TODO validations (length, format...)
+    // artist.id = parseInt(req.params.id, 10);
+
+    // models.artist.find(id);
```
Comment (LoicBrassart) [inline backend/src/controllers/CalendarController.js:60]: Dead code

**Review by LoicBrassart — APPROVED** [review]
SnE

File: backend/src/controllers/CalendarController.js (hunk 1 above)
Comment (ghost) [reply backend/src/controllers/CalendarController.js:60]: We will delete all the dead code on the final PR

**Pull request merged**